Fixed build with with automake 1.11 when it fails with
/bin/install: will not overwrite just-created `/usr/src/tmp/fluxbox-buildroot/usr/share/man/man1/fbrun.1' with `fbrun.1'
---
 doc/Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)
diff --git a/doc/Makefile.am b/doc/Makefile.am
index d440140..9da86e3 100644
--- a/doc/Makefile.am
+++ b/doc/Makefile.am
@@ -5,7 +5,7 @@ SUBDIRS = ru
 CLEANFILES = fluxbox.1 fluxstyle.1
 MAINTAINERCLEANFILES = Makefile.in
 man_MANS = fluxbox.1 fbsetroot.1 fbrun.1 fbsetbg.1 \
-	startfluxbox.1 fluxstyle.1 fbrun.1 fluxbox-keys.5 fluxbox-remote.1
+	startfluxbox.1 fluxstyle.1 fluxbox-keys.5 fluxbox-remote.1
 EXTRA_DIST=fluxbox.1.in fbsetroot.1 fbrun.1 fluxbox-keys.5 fluxbox-remote.1 startfluxbox.1 fbsetbg.1 \
 	fluxstyle.1.in CODESTYLE
 distclean-local:
